CHANGELOG.rst

```diff
@@ -24,6 +24,7 @@ Minor Changes
 Bugfixes
 --------
 
+- aws_s3 - Fix upload permission when an S3 bucket ACL policy requires a particular canned ACL
```

Comment:
CHANGELOG.rst is auto generated during a release.  Please create a fragment - https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs-how-to

Reply:
whoops - sorry about that. fixed